### ISSUE TITLE
Improve legends

### ISSUE THREAD

**Issue by adambraimbridge** [Financial-Times/next-beacon-v2#16]
![image](https://cloud.githubusercontent.com/assets/224547/12725878/b2f8d63e-c90c-11e5-94e9-cedc73dfa7b7.png)
Example: 'Members' might be better than 'count_unique'

**Comment by wheresrhys** [comment]
yep, I'll try to get kq to output something better. I think it's fine already for graphs with more than one dataset... it just breaks down for the simple case

**Comment by adambraimbridge** [comment]
Yeps, it's looking great for multi-dataset charts, awesome. (Though it does ignore the `label` column in the spreadsheets, which is incongruous —but not really a problem — e.g. the label for this chart is "Hustlers")

![image](https://cloud.githubusercontent.com/assets/224547/12895661/70597e1c-ce95-11e5-9041-1e797c8ab79d.png)
